core/client/fs/src/main/java/alluxio/client/file/cache/store/PageStoreDir.java

```diff
@@ -39,44 +38,45 @@ public interface PageStoreDir {
 
   /**
    * Create a list of PageStoreDir based on the configuration.
-   * @param conf AlluxioConfiguration
+   * @param options of cache manager
    * @return A list of LocalCacheDir
    * @throws IOException
    */
-  static List<PageStoreDir> createPageStoreDirs(AlluxioConfiguration conf)
+  static List<PageStoreDir> createPageStoreDirs(CacheManagerOptions options)
```

Comment:
can we pass in PageStoreOptions directly instead of CacheManagerOptions

Reply:
would expect create what pass what options

Reply:
good call, let me try